tests/Jellyfin.Networking.Tests/NetworkParseTests.cs

```diff
@@ -514,5 +514,25 @@ public void TestBindInterfaceOverrides(string source, string lan, string bindAdd
 
             Assert.Equal(intf, result);
         }
+
+        [Theory]
+        [InlineData("185.10.10.10,200.200.200.200", "79.2.3.4", false, true)] // whitelist
+        [InlineData("185.10.10.10", "185.10.10.10", false, false)] // whitelist
+        [InlineData("185.10.10.10", "79.2.3.4", true, false)] // blacklist
+        public void TestRemoteAccess(string addresses, string remoteIp, bool blacklist, bool denied)
```

Comment:
I know it matches the naming convention in the file but I'm trying to get us to use `MethodName_Scenario_ExpectedBehaviour` eg. `HasRemoteAccess_GivenNonEmptyWhitelist_AllowsOnlyIpsInWhitelist`. I would also like for this to be split into two tests, one for blacklist and one for whitelist.